src/OSPSuite.Core/Serialization/SimModel/Services/SimulationExportCreator.cs

```diff
@@ -328,7 +328,8 @@ private FormulaExport createFormulaExport(IUsingFormula usingFormula, IFormula f
                return _tableFormulaExportMapper.MapFrom(tableFormula);
 
             default:
-               return new ExplicitFormulaExport {Equation = formula.Calculate(usingFormula).ToString(NumberFormatInfo.InvariantInfo)};
+               // We need to use the G15 format specifier to revert to the behavior of .NET Framework for compatibility
+               return new ExplicitFormulaExport {Equation = formula.Calculate(usingFormula).ToString("G15", NumberFormatInfo.InvariantInfo)};
```

Comment:
This is the only place where we need this conversion?

Reply:
Can you add the link from the issue (this one https://devblogs.microsoft.com/dotnet/floating-point-parsing-and-formatting-improvements-in-net-core-3-0/#potential-impact-to-existing-code) to the comment?